src/index.ts

```diff
@@ -354,10 +354,7 @@ class SOOSSCAAnalysis {
     manifestFiles: Array<string>;
     hasMoreThanMaximumManifests: boolean;
   }): Promise<IUploadManifestFilesResponse> {
-    const formData = await analysisService.getAnalysisFilesAsFormData(
-      manifestFiles,
-      this.args.sourceCodePath,
-    );
+    const formData = await analysisService.getAnalysisFilesAsFormData(manifestFiles, process.cwd());
```

Comment:
`sourceCodePath` can be just a folder name. inside the method we have this line `const fileParts = analysisFile.path.replace(workingDirectory, "").split(Path.sep);` which replaces the sourceCodePath since it assumes is the full path and not a folder name.

Eg, if I have `sourceCodePath = manifests` and the full path of one found manifest is `repo/manifests/npm/package.json` the `fileParts` will end up being `repo/npm/package.json` reflecting a wrong path.

Reply:
we should enforce a full path if the code requires it by validating the value of the param